R/authenticate.R

```diff
@@ -8,15 +10,12 @@
 #'   
 #' @return sets environment variable with login information
 #' @export
-bwg_auth <- function(username, password) {
+bwg_auth <- function() {
   ## should check if it exists, and not rerun the function.
-  token_exists <- exists("token", envir = parent.frame())
-  
-  if (token_exists) stop("you've already got a token")
 
-  ## check inputs
-  assertthat::assert_that(is.character(username))
-  assertthat::assert_that(is.character(password))
+  username <- getPass::getPass("USERNAME: ")
```

Comment:
I don't think so.  If rstudioapi had a two-entry thing I'd recommend that but it doesn't.  I would argue that the unmasked user input is the path of least weirdness here (also, on terminal [Linux/OSX] you'll get both at the prompt, one masked one unmasked which is nice).